Source/WebCore/css/parser/CSSParser.h

```diff
@@ -63,7 +63,8 @@ class CSSParser {
     WEBCORE_EXPORT explicit CSSParser(const CSSParserContext&);
     WEBCORE_EXPORT ~CSSParser();
 
-    void parseSheet(StyleSheetContents&, const String&);
+    enum class IgnoreImportRules : bool { No, Yes };
+    void parseSheet(StyleSheetContents&, const String&, IgnoreImportRules = IgnoreImportRules::No);
```

Comment:
I think IgnoreImportRules could be a bit in CSSParserContext and you could avoid passing it around as argument.